Input text v2

Grebady opened the pull request.

Commit:
feat(input-text-v2) Добавлен utf-8 в html

Commit:
feat(input-text-v2) Добавлены файлы стилей и шрифтов

Commit:
feat(input-text-v2) Добавлен html шаблон

Commit:
feat(input-text-v2) Добавлены основные стили

Commit:
feat(input-text-v2) Добавлены иконки в спрайт

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 07-lection5/04-input/input.css:
```diff
@@ -0,0 +1,74 @@
+}
+
+.form-group__input::placeholder {
+    font-family: 'Inter', serif;
```
указываем только те стили которые отличают placeholder от value инпута

GSemikozov commented on 07-lection5/04-input/input.css:
```diff
@@ -0,0 +1,74 @@
+}
+
+.form-group__input:focus {
+    border: 1px solid var(--blue);
```
border-color

GSemikozov commented on 07-lection5/04-input/input.css:
```diff
@@ -0,0 +1,74 @@
+    cursor: not-allowed;
+}
+
+.form-group__icon {
```
по умолчанию скрыта

Commit:
fix(input-text-v2) Исправлены незначительные баги

1. Исправлены стили focus
2. Исправлен механизм показа иконок
3. Исправлены стили для placeholder

Commit:
fix(input-text-v2) label теперь не оборачивает input

GSemikozov commented on 07-lection5/04-input/input.css:
```diff
@@ -0,0 +1,68 @@
+    top: 1.0625rem;
+}
+
+.form-group__icon_success {
```
а в разметке почему-то это не используем

GSemikozov replied:
и не описаны состояния для блока - .form-group_success  + .form-group_error

Commit:
fix(input-text-v2) Добавлены стили для success модификатора

Commit:
fix(input-text-v2) Иконки меняются от модификатора блока

Grebady committed:
Merge branch 'master' into input-text-v2

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.
